### ISSUE TITLE
glyphSet is required to load glyphs

### ISSUE THREAD

**Issue by typemytype** [robotools/defcon#218]
```python
path = 'path/to/a/compressed.ufoz'  # a zipped ufo

from defcon import Font

f = Font(path)
print(f.layers.defaultLayer._glyphSet)
# without loading all glyphs
f.save()
# save will reset the _glyphSet object to None
print(f.layers.defaultLayer._glyphSet)
# but still not glyphs loaded
print(f.layers.defaultLayer._glyphs)
# but the font has some glyphName to be loaded
print(f.layers.defaultLayer.keys())
```

I dont know what the best option is: zipped ufozz are already slow as hell and now all glyphs needs to be loaded while reading for all al layers.... 

I understand the glyphSet object needs to be reset due to how zips work.
Is an immutable version possible?

This is a serious bug and makes ufoz not reliable...

**Comment by typemytype** [comment]
@anthrotype (to be sure you see this :) )

**Comment by anthrotype** [comment]
Oh yeah I saw it. Gimme some time to wrap my head around this

**Comment by typesupply** [comment]
Quick update: I've done some thinking and have a plan to fix this. I'll update everyone here when it is ready.

**Comment by benkiel** [comment]
Fixed in #220
